### PR TITLE
D-Bus api docs fixes

### PR DESCRIPTION
**Pull request by daandemeyer**
This finishes the review of all the D-Bus API docs.

### CODE REVIEW

File: man/org.freedesktop.systemd1.xml [hunk 1]
```diff
@@ -824,47 +827,46 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
...
       <literal>enabled-runtime</literal>, <literal>linked</literal>, <literal>linked-runtime</literal>,
       <literal>masked</literal>, <literal>masked-runtime</literal>, <literal>static</literal>,
-      <literal>disabled</literal>, <literal>invalid</literal>. <literal>enabled</literal> indicates that a
+      <literal>disabled</literal>, and <literal>invalid</literal>. <literal>enabled</literal> indicates that a
       unit file is permanently enabled. <literal>enable-runtime</literal> indicates the unit file is only
-      temporarily enabled, and will no longer be enabled after a reboot (that means, it is enabled via
+      temporarily enabled and will no longer be enabled after a reboot (that means, it is enabled via
       <filename>/run</filename> symlinks, rather than <filename>/etc</filename>). <literal>linked</literal>
-      indicates that a unit is linked into <filename>/etc</filename> permanently, <literal>linked</literal>
+      indicates that a unit is linked into <filename>/etc</filename> permanently. <literal>linked-runtime</literal>
```
Comment (keszybz) [inline man/org.freedesktop.systemd1.xml:840]: Nice catch!

File: man/org.freedesktop.systemd1.xml [hunk 2]
```diff
@@ -1073,27 +1075,27 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
...
       <constant>CLOCK_REALTIME</constant>/<constant>CLOCK_MONOTONIC</constant> microsecond timestamps when
       the process began and finished running the last time, or 0 if it never ran or never finished running;
       the PID of the process, or 0 if it has not run yet; the exit code and status of the last run. This
       field hence maps more or less to the corresponding setting in the service unit file but is augmented
       with runtime data.</para>
 
       <para><varname>LimitCPU</varname> (and related properties) map more or less directly to the
-      corresponding settings in the service unit files, however are set to 18446744073709551615 (i.e. -1) if
-      they aren't set.</para>
+      corresponding settings in the service unit files However, these are set to 18446744073709551615
```
Comment (keszybz) [inline man/org.freedesktop.systemd1.xml]: Dot missing here.

File: man/org.freedesktop.systemd1.xml [hunk 3]
```diff
@@ -1073,27 +1075,27 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
...
+      (i.e. -1) if they aren't set.</para>
```
Comment (keszybz) [inline man/org.freedesktop.systemd1.xml]: Hmm, maybe reword this, while at it. "these are set to ... if they aren't set" is a bit self-contradictory ;)

File: man/org.freedesktop.systemd1.xml [hunk 4]
```diff
@@ -1073,27 +1075,27 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
...
 
       <para><varname>Capabilities</varname> contains the configured capabilities, as formatted with
       <citerefentry project='man-pages'><refentrytitle>cap_to_text</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
       </para>
 
       <para><varname>SecureBits</varname>, <varname>CapabilityBoundingSet</varname>,
       <varname>MountFlags</varname> also correspond to the configured settings of the unit files, but are
-      encoded as the actual binary flag fields they are, rather than formatted as string.</para>
+      encoded as the actual binary flag fields they are rather than formatted as strings.</para>
```
Comment (keszybz) [inline man/org.freedesktop.systemd1.xml]: The original syntax is a bit weird, but without the comma it is harder to understand.

File: man/org.freedesktop.systemd1.xml (hunk 3 above)
Comment (daandemeyer) [reply man/org.freedesktop.systemd1.xml]: I went with "except that if they aren't set, their value is 18446744073709551615 (i.e. -1).". Maybe there's a better word than "is" though. Although using "is set to" is weird because of the self-contradiction.

File: man/org.freedesktop.systemd1.xml (hunk 4 above)
Comment (daandemeyer) [reply man/org.freedesktop.systemd1.xml]: Went with "of the unit files, but instead of being formatted as strings, they are encoded as the actual binary flags they are."
